iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/helplets.inl

```diff
@@ -155,8 +160,8 @@ inline bool doesEndWithPathSeparator(const string<StringCapacity>& name) noexcep
     {
         return false;
     }
-
-    char lastCharacter = name[name.size() - 1U];
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1: Basic numerical type of char shall be used
```

Comment:
I would change the justification to `Not used as an integer but as actual character`.

Reply:
@FerdinandSpitzschnueffler changed